python/benchmark/gen_data_distributed.py

```diff
@@ -448,6 +451,15 @@ def make_regression_udf(iter: Iterable[pd.DataFrame]) -> Iterable[pd.DataFrame]:
                 if noise > 0.0:
                     y += generator_p.normal(scale=noise, size=y.shape)
 
+                # Logistric Regression sigmoid and sample
+                if logistic_regression:
+                    if use_cupy:
+                        prob = 1 - 1 / (1 + cp.exp(-y))
```

Comment:
It seems y gets larger, prob will be smaller. Maybe not an issue. PR looks good.

Reply:
Yes if this prob of selecting 1, it should be 1/(1 + exp(-y))